toolkit/tools/internal/logger/log.go

```diff
@@ -245,3 +246,29 @@ func setHookLogLevel(hook *writerHook, level string) (err error) {
 	hook.SetLevel(logLevel)
 	return
 }
+
+// FormatWarningBox formats a message into a box with a border. The box is automatically sized to fit the longest line.
+// Each line will be centered in the box.
+func FormatWarningBox(message []string) []string {
+	var (
+		corners    = []string{"╔", "╗", "╚", "╝"}
```

Comment:
You don't use this as an array. So, it might be better to just give each symbol its own variable.

Reply:
done